README.md

```diff
@@ -14,37 +14,113 @@ All bugs, stories, and backlog for this project are managed through [Hyperledger
 in project IS (note that regular Indy tickets are in the INDY project instead...). Also, join
 us on [Hyperledger's Rocket.Chat](https://chat.hyperledger.org/) at #indy-sdk to discuss.
 
-## Building Indy SDK
+
+## How to install
+### Release channels
+There are three stable channels: 
+    
+* `master` - development builds for each push to master branch.
+* `rc` - release candidates.
+* `stable` - stable releases.
+
+Please refer to [release workflow](doc/release-workflow.md) for more details.
+### Install for Ubuntu based distro (Ubuntu 16.04)
+It is recommended to install packages with APT:
+    
+    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 68DB5E88
+    sudo add-apt-repository "deb https://repo.sovrin.org/sdk/deb xenial {release channel}"
+    sudo apt-get update
+    sudo apt-get install -y libindy
+
+See section "Release channels" for more details about {release channel}.  
```

Comment:
> See section "Release channels" for more details about {release channel}.  

Also unclear. Just write:

{release channel} must be replaced with master, rc or stable to define corresponded release channel.